RELEASE.md

```diff
@@ -54,3 +54,20 @@ git push upstream v1.2.0
 
 ### Post-Steps
 Once the tag has been pushed the release action should run automatically. You can view the progress [here](https://github.com/operator-framework/operator-controller/actions/workflows/release.yaml). When finished, the release should then be available on the [releases page](https://github.com/operator-framework/operator-controller/releases).
+
+
+## Backporting Policy
+
+Mainly critical issue fixes are backported to the most recent minor release.
```

Comment:
Suggested text `Important security fixes, critical bug fixes can be  back-ported to the most recent minor release` or something similar which would signal that it is not a agreement that all critical bug fixes will be backported.

Reply:
Thx. I changed a little following your suggestion
Feel free to check and let us know